varinfo/umm_var.py

```diff
@@ -71,6 +71,10 @@ def get_umm_var(var_info: VarInfoBase, variable: VariableBase) -> Dict:
 
     """
     variable_name = variable.full_name_path.lstrip('/')
+    if variable.is_science():
```

Comment:
I'd prefer to follow the pattern of the other optional attributes for `VariableType`, and put this conditional block into it's own function, something like `get_variable_type`. That way it keeps this function a bit cleaner, and the scope consistent. Also, it will be easier to iterate that function later ~~if~~ when we find cunning ways to identify other variable types and subtypes.

Reply:
* [ ] create `get_variable_type`